configs/ttalps_histogrammer_JPsiCR_config.py

```diff
@@ -141,7 +141,8 @@
     (collectionName , "logLxy"                , 2000   , -10    , 10     , ""  ),
     (collectionName , "dca"                   , 1000   , 0      , 20     , ""  ),
     (collectionName , "absCollinearityAngle"  , 500    , 0      , 5      , ""  ),
-    (collectionName , "invMass"               , 20000  , 0      , 200    , ""  ),
+    (collectionName , "invMass"               , 2000   , 0      , 200    , ""  ),
+    (collectionName , "logInvMass"            , 1000   , 0.1    , 100    , ""  ),
```

Comment:
Shouldn't this one (and all similar ones below) be in irregular histograms and have real log bins?

Oh wait, I see now how you fill them... why would you then decide to have bins from 0.1 to 100? I think it should be more like from -1 to 2 (meaning from 100 MeV to 100 GeV).